src/main/java/com/github/flaxsearch/resources/DocValuesResource.java

```diff
@@ -130,6 +136,68 @@ else if (dvtype == DocValuesType.SORTED_SET) {
         return response;
     }
 
+    @Path("/ordered")
+    @GET
```

Comment:
Can we make this ```/ordered/{field}``` to keep it in line with the other endpoints?  Plus it enforces that you need a field name for it to work.